package-res/ccc/core/base/chart/chart.data.js

```diff
@@ -96,6 +116,47 @@ pvc.BaseChart
 
         if(def.debug >= 3) this.log(this.data.getInfo());
     },
+ 
+
+    // CDF603
+    // Auxiliar function to initialize axes prior to the datums distribution
+    _initRolesAxes: function(){
+
+        hasMultiRole = this.visualRoles.multiChart.isBound();
+        this._initAxes(hasMultiRole);
+
+    },
+
+    // CDF603
+    // @virtual
+   _createScoringOptions: function(options) {
+         this._createSlidingWindow();
+         if(this.slidingWindow){
+            var sw = this.slidingWindow;
+            //override default scoring functions
+            this.data.score = function(datum) { sw.score.call( sw , datum ); }
+            this.data.select = function(allData, remove) { sw.select.call( sw , allData, remove ); }
+            return this;
+        }
+    },
+
+    // CDF603
+    // @virtual
+    _createSlidingWindow: function() {
+
+        var sw = this.options.slidingWindow;
+
+        if(this.slidingWindow){ this.slidingWindow.delete; }
+
+        if(sw) {
```

Comment:
Initially, the variable was for sliding window options, and now it switched to containing the classed sliding window options object...